app.js

```diff
@@ -3,27 +3,50 @@ const {
   getTopics,
   getEndPoints,
   getArticleById,
-  getAllArticles
+  getAllArticles,
+  getComments,
 } = require("./controller/controller")
+const error404 = (req, res, next) => {
+  res.status(404).send({ status: 404, msg: "Not Found" })
+}
```

Comment:
We shouldn't be hard coding this. It needs to take the custom error message and return that :) 
Please let us know if you aren't sure what I mean by this 👍